src/js/components/Grommet/stories/Background.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { grommet, Grommet, Box } from 'grommet';
+import { hpe } from 'grommet-theme-hpe';
+
+const Background = () => {
+  const themeColor = 'background-back';
+  const hexValue = '#DCD0FF';
+  const cssColor = 'gold';
+  return (
+    <Box gap="medium">
+      <Grommet>
+        <Box pad="medium">
+          <p>Grommet with no theme or background prop</p>
```

Comment:
Any reason we are not using Grommet's Paragraph?

Reply:
This should be paragraph, thank you.